test/test_fx.py

```diff
@@ -2289,7 +2289,8 @@ class TestOperatorSignatures(JitTestCase):
     @onlyCPU
     @ops(op_db, allowed_dtypes=(torch.float,))
     def test_get_torch_func_signature_exhaustive(self, device, dtype, op):
-        known_no_schema = {'stack', 'hstack', 'vstack', 'dstack', 'repeat', '__getitem__', 'linalg.multi_dot'}
+        known_no_schema = {'stack', 'hstack', 'vstack', 'dstack', 'repeat', '__getitem__', 'linalg.multi_dot',
```

Comment:
Is it obvious why this doesn't have a schema? Is it because it doesn't have a function variant?

Reply:
Can `tensordot` be added to this list as well?

Reply:
Perhaps `tensordot` should also be added in this list, in order to resolve #54574

But then it wouldn't be clear why they don't have schemas. eg. `tensordot` is a functional, but others aren't.

Reply:
Luckily in this case it seems like it was the lack of a function variant that was causing the issue cc @Chillee 

For now if we hit an issue with this test I agree we should extend this list

Longterm we should provide metadata on an opinfo and have that test check the metadata and know whether that metadata means the op has a schema for torch.fx or not

Reply:
Sorry. Forgot to reply here. But as @mruberry pointed the issue was resolved after adding the function variant. 

@imaginary-person Not sure as to why `tensordot` and others have trouble with it.